src/containers/my-cooperations/my-cooperations-details/MyCooperationsDetails.tsx

```diff
@@ -0,0 +1,142 @@
+import { useCallback } from 'react'
+import { useTranslation } from 'react-i18next'
+import { useNavigate, useParams } from 'react-router-dom'
+
+import DesignServicesIcon from '@mui/icons-material/DesignServices'
+import Avatar from '@mui/material/Avatar'
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import DoneIcon from '@mui/icons-material/Done'
+import PersonIcon from '@mui/icons-material/Person'
+import MessageIcon from '@mui/icons-material/Message'
+
+import useAxios from '~/hooks/use-axios'
+import { cooperationService } from '~/services/cooperation-service'
+import SubjectLevelChips from '~/components/subject-level-chips/SubjectLevelChips'
+import AppButton from '~/components/app-button/AppButton'
+import ShowMoreCollapse from '~/components/show-more-collapse/ShowMoreCollapse'
+import Loader from '~/components/loader/Loader'
+import { defaultOffer } from '~/containers/my-cooperations/my-cooperations-details/MyCooperationsDetails.constants'
+
+import { ButtonVariantEnum, SizeEnum, UserRoleEnum } from '~/types'
+import { style } from '~/containers/my-cooperations/my-cooperations-details/MyCooperationsDetails.styles'
+import { createUrlPath } from '~/utils/helper-functions'
+import { authRoutes } from '~/router/constants/authRoutes'
+import { useChatContext } from '~/context/chat-context'
+
+const MyCooperationsDetails = () => {
+  const { t } = useTranslation()
+  const { id = '' } = useParams()
+  const navigate = useNavigate()
+  const { setChatInfo } = useChatContext()
+
+  const getDetails = useCallback(
+    () => cooperationService.getCooperationById(id),
+    [id]
+  )
+
+  const { response, loading } = useAxios({
```

Comment:
we add some names for response.
here better use rewriting it `response: detailsResponse` and also can make it for `loading`